fda/grid.py

```diff
@@ -543,91 +547,151 @@ def concatenate(self, other):
                          self.dataset_label,
                          self.axes_labels)
 
-    def _set_labels(self, ax):
+    def _set_labels(self, fig):
```

Comment:
This function previously could accept either `matplotib.pyplot` or an axes object, IIRC. Can it accept those now?

Reply:
Previously it could only accept an axes object, now it can only accept a figure object since the purpose of this change is to expand it to more dimensions. There are as many axes objects as dimensions of the image, so if an axes object was accepted, the labels should be passed as arguments to avoid confusion.